templates/contribute/index.md

```diff
@@ -41,7 +41,7 @@ To ensure this, we only commit changes to `master` that have passed CI tests, an
 
 While you're working on a new contribution to `mathlib`, you should do this on a different branch.
 It's okay to do this in your own fork of the `mathlib` repository,
-or you can introduce yourself on Zulip and ask for write access to non-`master` branches of the mathlib repository.
+or you can introduce yourself on Zulip and ask for write access to non-`master` branches of the mathlib repository, you can either make your own thread to introduce yourself, or ask for access in [this topic](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/github.20permission).
 Please include your GitHub username in your request.
```

Comment:
Can we encourage people to put their github username in the Zulip profile too?

Reply:
What do you mean? The sentence after encourages them to add their username to the request, do you mean that or something else?

Reply:
I think Eric means "make it so that you can see someone's github userid on Zulip just by clicking on their name" and I was about to respond "but I don't recall being ever told to do this" but then I tried it on myself and indeed my github userid is visible.

Reply:
Yeah that sounds reasonable